packages/k8s.contrib.crd/generate.pkl

```diff
@@ -57,48 +57,76 @@
 module org.json_schema.contrib.generate
 
 import "pkl:yaml"
+import "pkl:semver"
 import "@deepToTyped/deepToTyped.pkl"
 import "@uri/URI.pkl"
+
 import "internal/ModuleGenerator.pkl"
 
+/// The version of the Pkl Kubernetes package to import.
+///
+/// This property is not used if [k8sImportPath] is set directly.
+k8sVersion: String(semver.isValid(this)) = "1.0.1"
 
+/// The base path to use for the Kubernetes imports.
+///
+/// Examples:
+/// ```
+/// // Change the version
+/// k8sImportPath = "package://pkg.pkl-lang.org/pkl-k8s/k8s@1.0.0#"
+///
+/// // Use dependency notation, assuming the dependency is called `@k8s`.
+/// k8sImportPath = "@k8s"
+/// ```
+k8sImportPath: String = "package://pkg.pkl-lang.org/pkl-k8s/k8s@\(k8sVersion)#"
```

Comment:
Added these two properties so that the Kubernetes version can be changed easily.